src/service/report-api/acceptance-tests/FamilyHubs.Report.Api.AcceptanceTests/Tests/Stories/ServiceSearchesForAnLaOrgTests.cs

```diff
@@ -1,10 +1,12 @@
+using System.Diagnostics.CodeAnalysis;
 using System.Net;
 using FamilyHubs.Report.Api.AcceptanceTests.Tests.Steps;
 using TestStack.BDDfy;
 using Xunit;
 
 namespace FamilyHubs.Report.Api.AcceptanceTests.Tests.Stories;
 
+[SuppressMessage("Usage", "xUnit1026:Theory methods should use all of their parameters")]
```

Comment:
Just curious on the warning suppression 🙂

Reply:
OK so one of the tests below has an unused parameter (role) but removing it means the test loses context. I could add a comment but I think Tina/Dami need to review the tests since we've added tighter rules on code validation